geoprocessing/src/utils/file.utils.ts

```diff
@@ -0,0 +1,27 @@
+/**
+ * File Utils
+ */
+
+import { MulterOptions } from '@nestjs/platform-express/multer/interfaces/multer-options.interface';
+import { Request } from 'express';
+import { diskStorage } from 'multer';
+const tempDirectory = require('temp-dir');
```

Comment:
why not import?

Reply:
Fixed [here](https://github.com/Vizzuality/marxan-cloud/pull/161/commits/47b6937375bcb9d4aeef04854d80b4d8be26ad29)